Dockerfile.run

```diff
@@ -1,13 +1,14 @@
+FROM alpine:3.4
 
-FROM    alpine:3.4
-RUN     apk -U add \
-            python \
-            py-pip
 
-COPY    requirements.txt /code/requirements.txt
-RUN     pip install -r /code/requirements.txt
+RUN apk update && apk add --no-cache \
+    curl openssl ca-certificates \
+    && curl -L https://github.com/docker/compose/releases/download/1.8.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose \
```

Comment:
This image is build from `script/build/image`.

Ya, we should use `COPY dist/docker-compose-Linux /usr/local/bin/docker-compose`, the problem is that we `.dockerignore` the `dist/` directory because we don't want it in the context normally. It can be quiet large and slows down other build steps.

It's too bad that `docker build` doesn't support a flag to set the `.dockerignore` file.

I think we can work around this by doing a copy of the files in `script/build/image` to a temp directory and building it from there.